src/qt/clientmodel.cpp

```diff
@@ -336,3 +337,15 @@ void ClientModel::unsubscribeFromCoreSignals()
     uiInterface.NotifyBlockTip.disconnect(boost::bind(BlockTipChanged, this, _1, _2, false));
     uiInterface.NotifyHeaderTip.disconnect(boost::bind(BlockTipChanged, this, _1, _2, true));
 }
+
+bool ClientModel::getProxyInfo(std::string& ip_port) const
```

Comment:
Nit, some may prefer `std::pair<bool, std::string> ClientModel::getProxyInfo() const;`, both work for me.

Reply:
Gonna go with what is currently implemented